### ISSUE TITLE
Link zur Installationsanleitung 

### ISSUE THREAD

**Issue by jan4401** [motom001/DoorPi#190]
Hi,
danke für das tolle Projekt. 
Beim Durchsehen ist mir aufgefallen, dass der Link zu den Installationsanleitungen in der README nicht funktioniert:
Die Installationen werden hier beschrieben -> https://www.doorpi.org/forum/lexicon/lexicon/6-installation/ -> Seite nicht gefunden

Grüße

**Comment by Fruchtzwerg94** [comment]
Ja, wirklich schade. Eine kurze Aktualisierung des Links hier wäre wirklich klasse - finden kann ich nämlich nichts.

**Comment by dvorpahl** [comment]
outdated .. :-D

**Comment by Pampersjoe** [comment]
Hallo zusammen, bin sei neuestem Admin im Forum und ja... das wurde etwas schleifen gelassen... ABER =) es wird weiter gehen... Installationsanleitung erst einmal hier zu finden: https://www.doorpi.org/forum/index.php?board/41-installation/

Auch wenns sicher zu spät ist.
